tests/data/insert-sample-data-in_db.js

```diff
@@ -9,8 +9,11 @@ VALUES(6,1,'Welcome to the new world',NOW(),NOW())`;
 export const INSERT_USER = `
 INSERT INTO users ("id","firstName","lastName","email","username","password","gender","address","role","isVerified","createdAt","updatedAt")
 VALUES(15,'updator','updator',updator@gmail.com,'updator','Updator@0000','male','kigali','requester',true,NOW(),NOW())`;
-export const INSERT_REQUEST = `
```

Comment:
Why did you delete this variable?

Reply:
i had to change how queries where written because the erd has changed